src/pages/Curation/components/Accordion/index.tsx

```diff
@@ -54,6 +65,42 @@ const Accordion = () => {
   const [checkedList, setCheckedList] = useState<Array<string>>([]);
   const [emotionsCheckedValue, setEmotionsCheckedValue] = useState<string>('');
 
+  const handleScroll = () => {
+    if (window.scrollY) {
+      const el = document.querySelector('nav');
+      el?.classList.remove('_wrapperHidden_quhlm_23');
```

Comment:
해당 class는 css module에 의해 생성된 className입니다. quhlm_23은 해시 값인데 css 내용이 바뀌면 해시 값도 변경(content hash)되게 됩니다. 그러므로 직접 값을 넣는 것이 아니라 import 후에 참조하는 방식으로 처리해야할 것입니다.

```suggestion
import bottomNavStyles from '@/components/BottomNav/style.module.scss';
// ...

      const el = document.querySelector('.' + bottomNavStyles.wrapperHidden);
      el?.classList.remove(bottomNavStyles.wrapperHidden');
```

스크롤을 내리더라도 bottomNav가 보이도록 하는 로직인 것 같은데 직접 className을 참조하는 방식은 그렇게 좋은 방식이 아닌 것 같군요. 우선은 이렇게 처리하고 시간이 된다면 개선하는게 좋을 것 같습니다.

Reply:
아하 저런 방법이 있다는걸 처음 알았네요! 감사드립니다~!